man/subscribe.Rd

```diff
@@ -0,0 +1,44 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/subscribe.R
+\name{subscribe}
+\alias{subscribe}
+\title{Subscribe to streaming market data}
+\usage{
+subscribe(securities, fields, fun, options = NULL, identity = NULL,
+  con = defaultConnection())
+}
+\arguments{
+\item{securities}{A character vector with security symbols in
+Bloomberg notation.}
+
+\item{fields}{A character vector with Bloomberg query fields.}
+
+\item{fun}{An R function to be called on the subscription data.}
+
+\item{options}{An optional named character vector with option
+values. Each field must have both a name (designating the option
+being set) as well as a value.}
+
+\item{identity}{An optional identity object.}
+
+\item{con}{A connection object as created by a \code{blpConnect}
+call, and retrieved via the internal function
+\code{defaultConnection}.}
+}
+\value{
+This function always returns NULL.
+}
+\description{
+This function uses the Bloomberg API to stream live market data
+}
+\examples{
+\dontrun{
+  subscribe(securities=c("/ticker/TYZ5 Comdty","/cusip/912810RE0@BGN"),
```

Comment:
from blpapi_subscriptionlist.h

you think I should add some of this documentation to the man page?

// STRUCTURE OF SUBSCRIPTION STRING
// ---------------------------------
// The simplest form of a subscription string is a *fully* *qualified*
// subscription string, which has the following structure:
//
// "//blp/mktdata/ticker/IBM US Equity?fields=BID,ASK&interval=2"
//  \-----------/\------/\-----------/\------------------------/
//        |          |         |                  |
//     Service    Prefix   Instrument           Suffix